cloud_edition/flexibility_mode/docs/disk_usage.rst

```diff
@@ -4,14 +4,14 @@ Disk Usage Management
 Each instance has its own dedicated disk:
 
 - 350GB for a production instance
-- 150GB for a sandbox instance
+- 350GB for a sandbox instance
```

Comment:
I rather prefer that we remove this block in order to not talk about specific size at all.
This value is a default one defined by the product and it could also be a different one when customer had an up-sale.
So not really relevant.

Reply:
Understood! :+1: